src/cli.js

```diff
@@ -214,7 +227,30 @@ async function signInToHeroku() {
   }
 
   log.info('Successfully logged into Heroku!');
-  return herokuCredentials;
+  // Create the app on Heroku
+  let appName = await utils.heroku.createApp(
+    configs.projectConfigurations.projectName,
+    herokuCredentials.apiKey
+  );
+
+  if (appName && appName.includes('Delete some apps or add a credit card to verify your account.')) {
+    log.info('You\\\'ve reached the limit of 5 apps for unverified accounts. Delete some apps or add a credit card to verify your account.');
+    disableHeroku = true;
+    return {
+      herokuCredentials,
+      appName: configs.projectConfigurations.projectName,
+      disableHeroku
+    };
+  }
+
+  while (appName && appName.includes('Name is already taken')) {
```

Comment:
Is there a graceful way for the user to break out of this while loop? E.g. "Enter SKIP to continue without setting up Heroku"

Reply:
#127 Created issue for this one